modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -287,17 +288,18 @@ def event_time(evt_message, node):
 
         return datetime.strptime(match.group(), "[%Y-%m-%d %H:%M:%S,%f]") if match else None
 
-    def get_event_time_on_node(self, node, log_pattern, from_the_beginning=True, timeout=15):
+    @staticmethod
```

Comment:
What a value this change brings? A suppose that not at all.

Reply:
fixed